model/src/main/kotlin/utils/OrtResultExtensions.kt

```diff
@@ -26,6 +26,16 @@ import org.ossreviewtoolkit.model.config.PathExclude
 
 import java.util.SortedSet
 
+/**
+ * Return a map of concluded licenses for each package [Identifier] that has a concluded license.
+ */
+fun OrtResult.collectConcludedLicenses(omitExcluded: Boolean = false): Map<Identifier, List<String>> =
+    getPackages(omitExcluded)
+        .filter { it.pkg.concludedLicense != null }
+        .associate {
+            Pair(it.pkg.id, it.pkg.concludedLicense?.licenses().orEmpty())
```

Comment:
That's not possible, `SpdxExpression` does not support parsing empty strings.

Reply:
My point was actually with the current signature it is not possible to represent the case where the license was concluded to SPDX.NONE.

Reply:
Why not? I would expect that `licenses()` just returns `listOf("NONE")` in this case.

Reply:
Hm, looking into the `SpdxExpression` it seems NONE cannot be represented - Did I miss anything?
If this is true we would not need to handle it here.

Reply:
There is a constant for `SpdxLicense.NONE`. If it is used in a concluded license it would be handled as a license id and be in this list, because concluded licenses are parsed using `Strictness.ALLOW_ANY`. If we want to handle this differently it should probably implemented in the `licenses()` function.